### ISSUE TITLE
Custom Dataset?

### ISSUE THREAD

**Issue by Respaired** [cantabile-kwok/vec2wav2.0#3]
Hi. thank you for making this work.

I wanted to ask how should the pipeline be changed to train this on a different, non-English & non-Chinese dataset? for instance, I'm confused about how spk2utt or utt2spk was originally created.

**Comment by cantabile-kwok** [comment]
Vocoders are language agnostic, for dataset metadata it is only required to have a `wav.scp` file (I have just updated the `data_prep.md` instruction to remove the confusion that `utt2spk,spk2utt` are needed as well). Obtaining this metadata is dataset-dependent though, because it relates to how the dataset is organized and stored. Basically these are the general steps:

1. Assume your dataset is organized into utterances (sentences), then there should be a unique utterance ID assigned for each one. 
2. Traverse through the dataset and record each utterance ID with its corresponding wav location, and save to `wav.scp`.

In short, utt2spk and spk2utt are not needed for training this vocoder. They are present in the downloaded zip file but that doesn't matter.

**Comment by Respaired** [comment]
I see, so utt2spk and spk2utt isn't needed and only the wav.scp matters. 
that clears it up! thank you.
